web/spec/app/controllers/drivingSchoolCtrlSpec.js

```diff
@@ -0,0 +1,91 @@
+var DrivingSchool = require(`${process.env.PWD}/app/models/DrivingSchool`);
```

Comment:
Ah, so that is what you were talking about that last time regarding the syntax needed to determine the current working directory!